addons/website/static/src/components/dialog/edit_menu.scss

```diff
@@ -1,16 +0,0 @@
-.ui-autocomplete.o_edit_menu_autocomplete {
-    max-height: 50vh;
-    overflow-y: auto;
-    overflow-x: hidden;
-    .ui-menu-item {
-        padding: 0;
-        > .ui-state-active {
-            border: none;
-            font-weight: normal;
-            margin: 0;
-        }
-    }
-    .fw-bold {
-        font-weight: 700 !important;
-    }
```

Comment:
You need to move this also with the new code, otherwise you break the auto complete in enterprise, I was not sure if you investigated if this was useful or not, but it is..
It is coming from https://github.com/odoo/odoo/commit/06516329a00c4554b7dd9f68f51b7d69ca81c813#

Without it, the enterprise section are broken:
Enterprise:
![image](https://user-images.githubusercontent.com/30048408/194062347-dd7c38a3-0765-4f75-8923-1fb615f1cff7.png)
Community:
![image](https://user-images.githubusercontent.com/30048408/194062404-8df5b938-e8ea-4c22-94e5-34b4148523ee.png)

Reply:
ok thanks, i forgot that part

Reply:
Actually the issue was that the autocomplete font family was not "Roboto" like the elements around but "Arial".
And Arial "font-weight: 500" has no effect... So instead of fixing it by changing 500 to 700, the correct solution is to also have Roboto for the automplete

Reply:
Unless you are on windows, it should not be Roboto but [system fonts everywhere ](https://github.com/odoo/odoo/commit/9f8f8f2ef88aee746819469c918b2d31ec29d031)now. If it's a more global backend problem, maybe it's worth reporting it SRI / framework. Meanwhile, I am sure we can stay with the bold solution.

Reply:
@qsm-odoo qsm ok I change it, I will talk to SRI about it.